README.md

```diff
@@ -39,8 +39,6 @@ GitHub issue for this : https://github.com/opsgenie/opsgenieclient/issues/34
 * User API
 * Notification Rule API
 * Notification Rule Step API
-* Team API
```

Comment:
Why did we remove it from the Readme?

Reply:
This is the list of APIs which are not supported for now but as we are providing support for v2 version now and they'll be functional with this change, so we should remove them from this list.

Reply:
Understood